packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -101,9 +101,16 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
   const mergedTextContainerStyle = mergeStyles(textContainerStyle, styles?.textFieldContainer);
   const mergedTextFieldStyle = concatStyleSets(textFieldStyle, {
     fieldGroup: styles?.textField,
-    errorMessage: styles?.systemMessage
+    errorMessage: styles?.systemMessage,
+    suffix: {
+      backgroundColor: 'transparent',
+      // Remove empty space in the suffix area when adding newline-style buttons
+      display: props.inlineChildren ? 'flex' : 'contents'
```

Comment:
nit: Seems like 10px padding on left and right is a bit much
![image](https://user-images.githubusercontent.com/79475487/236566856-e7bbb80b-51cf-49aa-99c7-9b670b19f4ba.png)
I would add style padding: '0 0.25rem' here in the suffix `IStyle` so there is less of a padding on the left and right like so:
![image](https://user-images.githubusercontent.com/79475487/236567018-27d0cec2-f173-4c67-8a11-8795359fc1e8.png)
It's closer to what is in Figma:
![image](https://user-images.githubusercontent.com/79475487/236567352-d5314a7e-97ea-499f-b2ef-656712727df3.png)

Reply:
Excellent observation! Updated! This also reminds me to update the padding for the TextField. We previously had extra padding to create space for the button rendering. However, since we now render the button directly within the TextField suffix, I've removed that padding value. Please let me know if I've missed anything! Thanks!
|original|previous|update|
|-----------|-----------|-----------|
| ![Screenshot 2023-05-05 at 12 04 03 PM](https://user-images.githubusercontent.com/75647512/236878718-eba8d04a-12f0-4635-8be0-ddc9b87fa1c0.png) | ![Screenshot 2023-05-05 at 12 01 11 PM](https://user-images.githubusercontent.com/75647512/236878783-9e444f9f-0d5a-42ab-89e2-ab6cc7aaaf1f.png) | ![Screenshot 2023-05-08 at 9 32 28 AM](https://user-images.githubusercontent.com/75647512/236879155-f3fdf788-e8ce-4daa-8a20-d25b6030818c.png) |